src/pyramid/config/routes.py

```diff
@@ -40,12 +40,9 @@ def add_route(
         inherit_slash=None,
         **predicates
     ):
-        r""" Add a :term:`route configuration` to the current
-        configuration state, as well as possibly a :term:`view
-        configuration` to be used to specify a :term:`view callable`
-        that will be invoked when this route matches.  The arguments
-        to this method are divided into *predicate*, *non-predicate*,
-        and *view-related* types.  :term:`Route predicate` arguments
+        r""" Add a :term:`route configuration` to the current configuration state.
+        The arguments to this method are divided into *predicate* and
+        *non-predicate* types.  :term:`Route predicate` arguments
```

Comment:
I have found that some whitespace things are not changed by black but caught by flake8.  Not sure about docstrings; I am used to PEP 257 conventions so I format to 72 columns before I run linters.

Reply:
https://pylonsproject.org/community-coding-style-standards.html

I need to update some of the Coding Style standards since we started using black ourselves. I haven't audited that. Another todo for that project. Pyramid's linters are the current standard.